packages/template/__tests__/keywords/debugger.test.ts

```diff
@@ -4,7 +4,7 @@ import { DebuggerKeyword } from '../../-private/keywords';
 const debuggerKeyword = resolve({} as DebuggerKeyword);
 
 // Can be invoked as {{debugger}}
-emitContent(debuggerKeyword({}));
+emitContent(debuggerKeyword());
```

Comment:
The lack of extra empty objects in so many places really is nice. (Noticed repeatedly, just thought to comment on it on seeing this one!)